### PR TITLE
Update system recommendations

### PR DESCRIPTION
**Pull request by darshanasbg**
Updating $subject to be clear on what's the recommended configuration and what's the (recommended) minimum configuration in the doc:
https://is.docs.wso2.com/en/latest/setup/deployment-guide/

### CODE REVIEW

File: en/docs/setup/deployment-guide.md [hunk 1]
```diff
@@ -31,38 +31,13 @@ necessary system requirements and a compatible environment.
...
-</ul></td>
-</tr>
-<tr class="even">
-<td>Cassandra data nodes</td>
-<td><ul>
-<li>4 core processors</li>
-<li>8 GB RAM</li>
+<li>10 GB Disk Space</li>
 </ul>
+<p>Above recommended requirements can changed based on the expected concurrency & performance.</p></td>
```
Comment (gomathyk) [inline en/docs/setup/deployment-guide.md]: The above recommendations can change based on the ...

File: en/docs/setup/deployment-guide.md [hunk 2]
```diff
@@ -31,38 +31,13 @@ necessary system requirements and a compatible environment.
 </colgroup>
 <tbody>
 <tr class="odd">
-<td>Physical</td>
+<td></td>
```
Comment (gomathyk) [inline en/docs/setup/deployment-guide.md]: Why are we removing the section header?

Comment (darshanasbg) [reply en/docs/setup/deployment-guide.md]: As we don't have multiple sections anymore.

File: en/docs/setup/deployment-guide.md (hunk 1 above)
Comment (darshanasbg) [reply en/docs/setup/deployment-guide.md]: updated as mentioned. Thanks!

**Pull request merged**